src/Router.sol

```diff
@@ -173,6 +176,7 @@ contract Router is IRouter, EIP712, FeeGenerator {
         agent.execute{value: msg.value}(logics, tokensReturn);
     }
 
+    /// @dev Signature can be reused on any chain before the deadline
```

Comment:
```suggestion
    /// @dev Allow whitelisted signers to use custom fee rules and permit the reuse of the signature until the deadline
```

- Prefer to put @dev after @notice to follow the order in https://docs.soliditylang.org/en/v0.8.20/natspec-format.html#tags
- Provide the reason of the reuse

Reply:
fixed.